src/PowerShell/PowerShellManager.cs

```diff
@@ -60,7 +117,7 @@ internal void AuthenticateToAzure()
                 string.IsNullOrEmpty(applicationSecret) ||
                 string.IsNullOrEmpty(tenantId))
             {
-                _logger.Log(LogLevel.Warning, "Required environment variables to authenticate to Azure were not present");
+                _logger.Log(LogLevel.Warning, "Skip authentication to Azure via Service Principal. Environment variables for authenticating to Azure are not present.");
```

Comment:
Same as above, is `Warning` the right level to use here?
And, we don't write any messages in case the MSI environment variables are not present.

Reply:
Trace?